src/BenchmarksDriver/Program.cs

```diff
@@ -592,7 +579,7 @@ public static int Main(string[] args)
                     var p = command.Parameters;
                     p.AddWithValue("@DateTime", DateTimeOffset.UtcNow);
                     p.AddWithValue("@Scenario", scenario.ToString());
-                    p.AddWithValue("@Framework", framework.ToString());
+                    p.AddWithValue("@Framework", "Core");
```

Comment:
We should preserve the historical data, which means we cannot delete the column from SQL.  Your change is good for now.  After your change is merged, I will update the column in the live database to include a default value, and remove the column from the insert statement.